feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/domain/era/RelaychainStakingEraInteractor.kt

```diff
@@ -5,18 +5,18 @@ import io.novafoundation.nova.feature_staking_impl.domain.StakingInteractor
 import io.novafoundation.nova.feature_staking_impl.domain.staking.start.model.StartStakingEraInfo
 import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
 import kotlinx.coroutines.flow.Flow
-import kotlin.time.Duration.Companion.milliseconds
 
 class RelaychainStakingEraInteractor(
     private val stakingInteractor: StakingInteractor
 ) : StakingEraInteractor {
 
     override fun observeEraInfo(chain: Chain): Flow<StartStakingEraInfo> {
         return flowOf {
+            val eraDuration = stakingInteractor.getEraDuration()
             StartStakingEraInfo(
-                remainingEraTime = stakingInteractor.getRemainingEraTime().toLong().milliseconds,
                 unstakeTime = stakingInteractor.getLockupDuration(),
-                eraDuration = stakingInteractor.getEraDuration()
+                eraDuration = eraDuration,
+                firstRewardReceivingDuration = stakingInteractor.getFirstRewardReceivingDelay()
```

Comment:
Here we create eraTimeCalculator 3 times which result in about 15 consequent requests to node